aspnetcore/signalr/javascript-client.md

```diff
@@ -93,6 +93,17 @@ Use the [configureLogging](/javascript/api/%40aspnet/signalr/hubconnectionbuilde
 
 [!code-javascript[Logging levels](javascript-client/sample/wwwroot/js/chat.js?range=9-12)]
 
+## Reconnect Clients 
+
+The JavaScript client for SignalR doesn't automatically reconnect. As a result, you must write code that will reconnect your client manually. The following code demonstrates a typical reconnection approach:
```

Comment:
~~As a result, you must write code that will reconnect your client manually.~~
You must write code that manually reconnects your client.